tools/libdeflate/Makefile

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (C) 2022 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libdeflate
+PKG_VERSION:=1.12
+PKG_RELEASE:=1
```

Comment:
Can you use the new AUTORELEASE ?

Reply:
Simply PKGRELEASE:=$(AUTORELEASE) or how does that work?

Reply:
Yes simply that 

`PKG_RELEASE:=$(AUTORELEASE)`